### PR TITLE
[IMP] event(_sale), website_event_track: UX Refinement

### PR DESCRIPTION
**Pull request by nounoubensebia**
This commit aims at refining the UX for events, notably:

- create a helper for the magic button on the event_track form, because without
  this the user won't be able to understand what the button does.

- on stage update, the kanban state will be automatically set to "gray"
  beacause when changing stages it does not make sense to keep the previous
  kanban state especially when it is set to "green" or to "red".

- make the cost of the event registration product demo data lower then the sale
  price as it is not normal to have a sale price lower then the cost.

- change the style of the "discover all our events" marketing link in the event
  reminder email template in order to make it pop a bit more for the user as
  this is an important link

Task-2451125
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/event/models/event_event.py [hunk 1]
```diff
@@ -203,7 +203,7 @@ def _default_description(self):
     badge_innerright = fields.Html(string='Badge Inner Right')
     event_logo = fields.Html(string='Event Logo')
 
-    @api.depends('stage_id', 'kanban_state')
```
Comment (awa-odoo) [inline addons/event/models/event_event.py:206]: Why this change?

Since we use the stage_id to compute the label, I think it can stay there.
Looks like all other implementations of kanban_state_label specify stage_id in the depends as well.

File: addons/event/models/event_event.py [hunk 2]
```diff
@@ -455,6 +455,11 @@ def _compute_event_ticket_ids(self):
                 ]
             event.event_ticket_ids = command
 
+    @api.onchange('stage_id')
+    def _onchange_stage_id(self):
+        for event in self:
+            event.kanban_state = 'normal'
```
Comment (awa-odoo) [inline addons/event/models/event_event.py:461]: As a general rule, we try to avoid implementing behaviors with the "onchange" method, it tends to be deprecated over time.

Instead, I see in projets that the "write" method is overridden to handle the kanban_state reset when the task changes stage:

```py
if 'stage_id' in vals:
    ...
    # reset kanban state when changing stage
    if 'kanban_state' not in vals:
        vals['kanban_state'] = 'normal'
```

**Review by awa-odoo — COMMENTED** [review]
Hello @nounoubensebia 

Nice small UX task overall.
I left a few comments.

For your commit message:
> [IMP] event(_sale), website_event_track: UX Refinement 

Keep in mind that there should always be a verb after the ":" in the commit header.
So here it could be "[IMP] event(_sale), website_event_track: apply small interface tweaks" for example.

You could also rewrite the message and apply a bit better formatting like explained in previous review (up to 80 characters per line, ...).

Cheers!

File: addons/event/models/event_event.py [hunk 3]
```diff
@@ -501,6 +501,10 @@ def create(self, vals_list):
         return events
 
     def write(self, vals):
+        if 'stage_id' in vals:
+            # reset kanban state when changing stage
+            if 'kanban_state' not in vals:
```
Comment (awa-odoo) [inline addons/event/models/event_event.py:506]: You can probably merge the 2 conditions then:
`if 'stage_id' in vals and 'kanban_state' not in vals`

**Comment by tde-banana-odoo** [comment]
@robodoo r+
